src/routes/exercise-finished/ExerciseFinishedScreen.tsx

```diff
@@ -32,8 +40,16 @@ const MessageContainer = styled.View`
   margin-top: ${props => props.theme.spacings.sm};
 `
 const Message = styled(HeadingBackground)`
+  color: ${ExerciseUnlocked ? prop => prop.theme.colors.primary : prop => prop.theme.colors.background};
   text-align: center;
 `
+const Icon = styled.TouchableOpacity`
+  position: absolute;
+  padding-bottom: 250px;
+  padding-left: 300px;
+  width: ${wp('6%')}px;
```

Comment:
No need for wigth and height here, as they are passed directly in the rendering below. Please remove.